holoviews/plotting/plotly/plot.py

```diff
@@ -45,8 +65,9 @@ def initialize_plot(self, ranges=None):
 
 
     def update_frame(self, key, ranges=None):
-        return self.generate_plot(key, ranges)
-
+        plot = self.generate_plot(key, ranges)
+        PlotlyRenderer.trigger_plot_pane(self.id, self.state)
```

Comment:
Does this have to happen here or is the triggering that happens in ``.push`` enough? One thing I just realized is that the current ``push`` approach probably doesn't work for Layouts and I need to propagate the ``Plot._pane`` down to all child plots anyway.